task_sdk/src/airflow/sdk/api/client.py

```diff
@@ -445,6 +466,12 @@ def assets(self) -> AssetOperations:
         """Operations related to Assets."""
         return AssetOperations(self)
 
+    @lru_cache()  # type: ignore[misc]
+    @property
```

Comment:
Should this not use `cached_property` instead?

Reply:
My guess is that it didn't work, likely cos httpx.Client is slotted? That or I stupidly never tried it 🤷🏻

Reply:
Sure yeah, doesnt matter in this one